lesson1/Route.rb

```diff
@@ -0,0 +1,21 @@
+class Route
+  attr_reader :stations
+
+  def initialize(startStation, endStation)
```

Comment:
```suggestion
  def initialize(start_station, end_station)
```
тот же недочет
это не Java и не JS,  в руби пишут "снейк кейсом"